src/stylesheets/components/_toggles.scss

```diff
@@ -0,0 +1,35 @@
+/*--- Toggles ---*/
+
+.toggle--menu {
+  position: relative;
+  z-index: 2;
+  margin-bottom: $base-spacing-unit-small;
+}
+
+.has-js .toggle--menu > .block {
+  position: absolute;
+  left: 0;
+  width: 100%;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  margin-top: $base-spacing-unit-large;
+
+  &::before {
+    content: "";
+    position: absolute;
+    bottom: 100%;
+    left: ($base-spacing-unit-small * 6);
+    border: $base-spacing-unit-small solid transparent;
+    border-top: 0;
+    border-bottom-color: #fff;
+  }
+}
+
+body:not(.has-js) .toggle--menu > .block {
+  display: block;
+}
+
+/*--- Functional ---*/
+
+[data-toggle="content"]:not(.open) {
```

Comment:
I think we need rid ourselves of the `:not()` pseudo class by having classes for each state (`.open` or `.close`) that we can target

Reply:
Will also skip this for now as it's a larger project and will be better to treat as a separate project.